src/Federation/Resolvers/Entities.php

```diff
@@ -33,27 +28,49 @@ public function __invoke($root, array $args, GraphQLContext $context, ResolveInf
     {
         $results = [];
 
-        /** @var array<string, array<int, array<string, mixed>>> $groupedRepresentations */
+        $representations = $args['representations'];
+        $representationHashes = array_map('serialize', $representations);
```

Comment:
Is it in any way possible for the serialized result to give the same response twice? 🤔

Reply:
Nevermind, idiot question, we actually want that 😂